src/Umbraco.Core/Scoping/Scope.cs

```diff
@@ -235,6 +237,7 @@ public IUmbracoDatabase Database
                 try
                 {
                     _database.BeginTransaction(IsolationLevel);
+                    _database.Execute("SET LOCK_TIMEOUT 1800;");
```

Comment:
Perhaps should be part of SqlSyntax

Reply:
I 'think', this doesn't actually need to be here since the lock timeout call is made when actually creating a write lock when in sql server mode.